addons/account/tests/test_transfer_wizard.py

```diff
@@ -234,16 +234,15 @@ def test_transfer_default_tax(self):
                 }),
                 Command.create({
                     'account_id': self.payable_account.id,
-                    'balance': -460,
+                    'balance': -400,
                 }),
             ]
         })
         move_with_tax.action_post()
 
         self.assertRecordValues(move_with_tax.line_ids, [
             {'balance': 400, 'account_id': account_with_tax.id},
-            {'balance': -460, 'account_id': self.payable_account.id},
-            {'balance': 60, 'account_id': self.company_data['default_account_tax_purchase'].id},
```

Comment:
Even if there is explicitly an account with a tax set?

Then this whole test case doesn't make sense anymore